lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -219,14 +219,31 @@ extension CanvasKitExtension on CanvasKit {
   ) => _MakeImage(info, pixels.toJS, bytesPerRow.toJS);
 
   @JS('MakeLazyImageFromTextureSource')
-  external SkImage? _MakeLazyImageFromTextureSource(
+  external SkImage? _MakeLazyImageFromTextureSource1(
```

Comment:
Not sure if there's value in reusing the Dart SDK convention, but there the convention is for the number to refer to the argument count, so in this case it would be "2" and "3", instead of "1" and "2".